dist/access_check/check_sources.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import sys, os
+from layercparse import *
```

Comment:
Can we specify which members of the layercparse package we'd like to import here? 
("from xyz import *" feels like an anti-pattern)

Reply:
One inconvenience for me is the confusion about which are the imported library members used in this script. It also makes VSCode treat them as "undefined" - maybe it's because I did not set up a venv and install the package.

Reply:
Ok I changed the way the module is imported. I hope it helps.